faq.md

```diff
@@ -380,6 +380,24 @@ framework, we've decomposed complex widgets to pieces that separately implement
 the visual, interaction, and motion design. You can remix these widgets however
 you like to make your own custom widgets that have full range of expression.
 
+### Why would I want to share layout code across iOS and Android?
+
+More and more, we see mobile app layouts and designs converging
+to be more brand-driven and unified across platforms. For example,
+the "bottom nav bar" pattern can now be naturally found across iOS and Android.
```

Comment:
I suggest we swap the order of 'brand-driven' and 'unified across platforms' so that they are in the same order as the examples in the next two sentences.

Reply:
I'll reorder to match